docs/modules/ROOT/pages/advanced-how-tos/managing-multiple-versions.adoc

```diff
@@ -63,53 +62,101 @@ spec:
 
+The following kinds of resources are currentyl supported. Please do not attempt to create other kinds as that would prevent the template from being applied successfully.
+
+[cols="2,1,3"]
+|===
+|apiVersion |kind |Comments
+
+|appstudio.redhat.com/v1alpha1
+|Application
+|
+
+|appstudio.redhat.com/v1alpha1
+|Component
+|
+
+|appstudio.redhat.com/v1alpha1
+|ImageRepository
+|Every component should have an accompanying `ImageRepository` resource
+
+|appstudio.redhat.com/v1beta2
+|IntegrationTestScenario
+|Note the `v1beta2` API version. Make sure to use the right version when querying the cluster, otherwise important data may be missing from the resutls you get.
+
+|appstudio.redhat.com/v1alpha1
+|ReleasePlan
+|
+|===
+
+*Note:* When in doubt - the source of truth about what the YAML for your resources should look like is the up to date YAML you can find on the Konflux clusters. We try to keep the exaples here up to date, but with this being a static document there may be a gap between what you see here and what you should configure in tour system. It is recommended that you follow the process of creating the template by copying the YAML of the resrouces you have and cleaning it as describe blow, and not by copying the sample you see here.
+
 Here are specific examples for how to clean up and use the YAML for certain resource kinds:
 
-* For any kind of resource, the `namespace`, `creationTimestamp`, `generation`, `resourceVersion`, `uid` and `ownerReferences`, metadata fields should be removed as well as the `status` section.
+* For any kind of resource, the `namespace`, `creationTimestamp`, `generation`, `resourceVersion`, `uid`, `ownerReferences` and `finalizers` metadata fields should be removed as well as the `status` section.
 * For *Application* resources the `metadata.name` and `spec.displayName` fields should contain variable references.
 * For *Component* resources:
-** The following annotations should be removed:
-*** `build.appstudio.openshift.io/status`
+** The following deprecated annotations should be removed:
 *** `image.redhat.com/image`
 ** The `spec.containerImage` field should be removed.
 ** The following fields should probably contain variable references:
 *** `spec.application`
 *** `spec.componentName`
 *** `source.git.revision`
+** The `build.appstudio.openshift.io/status` annotation is in place to make the UI present the component's pipeline as customized. Its not mandatory if you can ignore the pipeline status being missrepresented in the UI. Note that this annotation's value needs to contain a reference to a merged PR that added the PaC pipeline files. There is no harm in having multiple components reference the same PR, as long as the `pac.component.appstudio.openshift.io/finalizer` finalizer is not added to the component's finalizers list.
```

Comment:
```suggestion
** The `build.appstudio.openshift.io/status` annotation makes the {ProductName} UI present the component's pipeline as customized. While it is not functionally required for component onboarding, omitting the annotation will result in a misrepresentation of this property in the UI. Note that this annotation's value needs to contain a reference to a merged PR that added the PaC pipeline files. There is no harm in having multiple components reference the same PR, as long as the `pac.component.appstudio.openshift.io/finalizer` finalizer is not added to the component's finalizers list.
```

Should the Note portion of this bullet be removed and placed elsewhere?

Reply:
Putting it elsewhere would place it out of context IMO.